qemu/tests/boot_nic_with_intel_iommu.py

```diff
@@ -18,7 +18,7 @@ def run(test, params, env):
     :param env: Dictionary with test environment.
     """
 
-    if utils_misc.get_cpu_vendor(verbose=False) != 'GenuineIntel':
+    if cpu.get_cpu_vendor_name() != 'GenuineIntel':
```

Comment:
Actually it will return different values than the original one, so have to update the pattern together.
```
:returns: string 'intel' or 'amd' or 'power7' depending on the current CPU architecture.
```

Reply:
Yes, you are right, updated, thanks.